lib/src/contextual_message.dart

```diff
@@ -18,7 +18,7 @@ class ContextualMessage {
   String message;
 
   /// Optional context map, used for including additional data.
-  Map<String, dynamic> context;
+  Map<String, dynamic>? context;
```

Comment:
```suggestion
  Map<String, dynamic> context;
```
I think we can make this non-nullable with a default in the constructor